resources/nuts-node/auth.js

```diff
@@ -1,21 +1,50 @@
-const config = require('../../util/config');
-const call   = require('./open-api-helper').call({
-  baseURL:    `http://${config.nuts.node}`,
-  definition: 'https://raw.githubusercontent.com/nuts-foundation/nuts-auth/master/docs/_static/nuts-auth.yaml'
-});
+const config = require('../../util/config')
+var FormData = require('form-data')
+const apiHelper = require('./open-api-helper')
+const definitionLocation = 'https://raw.githubusercontent.com/nuts-foundation/nuts-auth/0cdd505d38ac3387062437ffe25863ca2cc4a11a/docs/_static/nuts-auth.yaml'
+const call = apiHelper.call({
+  baseURL: `http://${config.nuts.node}`,
+  definition: definitionLocation
+})
 
 module.exports = {
-  createLoginSession:   async ()         => await call('createSession', null, loginContract()),
-  createSession:        async (contract) => await call('createSession', null, contract),
-  sessionRequestStatus: async (id)       => await call('sessionRequestStatus', id),
-  validateContract:     async (contract) => await call('validateContract', null, contract)
-};
+  createLoginSession: async () => await call('createSession', null, loginContract()),
+  createSession: async (contract) => await call('createSession', null, contract),
+  sessionRequestStatus: async (id) => await call('sessionRequestStatus', id),
+  validateContract: async (contract) => await call('validateContract', null, contract),
+  createJwtBearerToken: async (context) => await call('createJwtBearerToken', null, context),
+  createAccessToken: async (baseUrl, jwtBearerToken) => {
+    // createAccessToken is a bit weird since it follows the OAuth spec and needs a FormData object instead of a plain json document
+    // The baseUrl is added since the createAccessToken is usually performed on an other Nuts node than your own
```

Comment:
Dit moet je me toch effe uitleggen hoor... je gaat toch geen calls op de Nuts node van de ander doen..?

Reply:
Toch wel :) zie comment in de hoofd thread.

Reply:
Check ;)